pkg/storage/sqlstorage/log.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"github.com/numary/go-libs/sharedlogging"
```

Comment:
linter should tell you to move this lower with the other github/ imports i think